google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/StreamingSubscriberConnection.java

```diff
@@ -216,8 +216,10 @@ private void initialize() {
             .setSubscription(subscription)
             .setStreamAckDeadlineSeconds(60)
             .setClientId(clientId)
-            .setMaxOutstandingMessages(flowControlSettings.getMaxOutstandingElementCount())
-            .setMaxOutstandingBytes(flowControlSettings.getMaxOutstandingRequestBytes())
+            .setMaxOutstandingMessages(
```

Comment:
Is zero ignored as a value or is it treated truly as allow zero messages/bytes to be outstanding?

Reply:
If the value is zero, there will be no limit enforced:

https://github.com/googleapis/java-pubsub/blob/master/proto-google-cloud-pubsub-v1/src/main/java/com/google/pubsub/v1/StreamingPullRequest.java#L2073